projects/output-and-error-handling/main.go

```diff
@@ -0,0 +1,102 @@
+package main
+
+import (
+	"errors"
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+	"strconv"
+	"time"
+)
+
+type RequestError struct {
+	SleepTime int
```

Comment:
I'd probably use a `time.Duration` here rather than an `int` - with an `int` it's unclear what the units are here (Milliseconds? Seconds? Minutes? Days?) - a `time.Duration` has units built in.